cluster/src/main/java/io/servicefabric/cluster/Cluster.java

```diff
@@ -217,4 +217,15 @@ private void updateClusterState(State expected, State update) {
 		boolean stateUpdated = state.compareAndSet(expected, update);
 		checkState(stateUpdated, "Illegal state transition from %s to %s cluster state. Expected state %s.", state.get(), update, expected);
 	}
+
+	/**
+	 * check if a given member is a local member return true in case the ClusterMember is local to the cluster instance
+	 * @param ClusterMember
+	 * @return true if this cluster memeber is a local cluster memeber
+	 * @author ronen hamias
+	 */
+	@Override
+	public boolean isLocalMember(ClusterMember member) {
+		return membership().localMember().endpoint().equals(member.endpoint());
```

Comment:
Lets move this method on IClusterMembership interface. So from cluster it will be:

cluster.membership().isLocalMember(member)